StoreApp.BLL/Services/OrderDetailService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using System.Collections.Generic;
+
+namespace StoreApp.BLL.Services
+{
+    class OrderDetailService : IOrderDetailService
```

Comment:
The default modifier for classes is internal. Are you sure that you want to leave this default setting?
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers

Reply:
fixed